src/session.cpp

```diff
@@ -34,8 +34,6 @@ POSSIBILITY OF SUCH DAMAGE.
 #include "libtorrent/extensions/ut_pex.hpp"
 #include "libtorrent/extensions/ut_metadata.hpp"
 #include "libtorrent/extensions/smart_ban.hpp"
-#include "libtorrent/session.hpp"
```

Comment:
just a comment: I'm in favor of cleanup as much as possible, but in my opinion, the `.cpp` should include the corresponding `.hpp` if only for coding style.